--- xarrayutils-2.0.1.orig/setup.py	2024-05-15 19:15:16.000000000 +0200
+++ xarrayutils-2.0.1/setup.py	2024-05-20 20:00:48.414693383 +0200
@@ -1,5 +1,4 @@
 import os
-import versioneer
 from setuptools import setup, find_packages
 
 here = os.path.dirname(__file__)
@@ -48,8 +47,7 @@
     install_requires=install_requires,
     extras_require=extras_require,
     python_requires=">=3.9",
-    version=versioneer.get_version(),
-    cmdclass=versioneer.get_cmdclass(),
+    version="2.0.1",
     long_description=long_description,
     long_description_content_type="text/markdown",
 )
